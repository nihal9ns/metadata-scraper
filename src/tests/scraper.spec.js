const { scrapeMetaData } = require("../scraper");

describe("TEST SCRAPER", () => {
  it("Should return scraped meta data", async () => {
    const url = "https://producthunt.com";

    const result = await scrapeMetaData(url);

    expect(result).toMatchSnapshot();
  });
});
