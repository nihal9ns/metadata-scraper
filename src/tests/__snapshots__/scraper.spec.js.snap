// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEST SCRAPER Should return scraped meta data 1`] = `
Object {
  "description": "Product Hunt is a curation of the best new products, every day. Discover the latest mobile apps, websites, and technology products that everyone's talking about.",
  "ogImage": "https://api.url2png.com/v6/P5329C1FA0ECB6/790272390317dc724643b1ca88f5da6e/png/?url=https%3A%2F%2Fwww.producthunt.com%2F",
  "ogTitle": "Product Hunt – The best new products in tech.",
  "title": "Product Hunt – The best new products in tech.",
}
`;
